src/main/java/org/opentripplanner/graph_builder/module/PruneFloatingIslands.java

```diff
@@ -78,11 +108,194 @@ public void buildGraph(
+            if (islandLog != null) {
+                WriteNodesInSubGraph(island, islandLog, hadRemoved);
+            }
+        }
+        if (graph.removeEdgelessVertices() > 0) {
+            LOG.info("Removed edgeless vertices after pruning islands");
+        }
+    }
+
+    private static void depedestrianizeOrRemove(
+            Graph graph,
+            Subgraph island,
+            DataImportIssueStore issueStore
+    ) {
+        //iterate over the street vertex of the subgraph
+        for (Iterator<Vertex> vIter = island.streetIterator(); vIter.hasNext();) {
+            Vertex v = vIter.next();
+            Collection<Edge> outgoing = new ArrayList<Edge>(v.getOutgoing());
+            for (Edge e : outgoing) {
+                if (e instanceof StreetEdge) {
+                    StreetEdge pse = (StreetEdge) e;
+                    StreetTraversalPermission permission = pse.getPermission();
+                    permission = permission.remove(StreetTraversalPermission.PEDESTRIAN);
+                    permission = permission.remove(StreetTraversalPermission.BICYCLE);
+                    if (permission == StreetTraversalPermission.NONE) {
+                        graph.removeEdge(pse);
+                    } else {
+                        pse.setPermission(permission);
+                    }
+                }
+            }
+        }
+
+        for (Iterator<Vertex> vIter = island.streetIterator(); vIter.hasNext();) {
+            Vertex v = vIter.next();
+            if (v.getDegreeOut() + v.getDegreeIn() == 0) {
+                graph.remove(v);
+            }
+        }
+        //remove street conncetion form
+        for (Iterator<Vertex> vIter = island.stopIterator(); vIter.hasNext();) {
+            Vertex v = vIter.next();
+            Collection<Edge> edges = new ArrayList<Edge>(v.getOutgoing());
+            edges.addAll(v.getIncoming());
+            for (Edge e : edges) {
+                if (e instanceof StreetTransitStopLink || e instanceof StreetTransitEntranceLink) {
+                    graph.removeEdge(e);
+                }
+            }
+        }
+        issueStore.add(new GraphConnectivity(island.getRepresentativeVertex(), island.streetSize()));
+    }
+
+    private static Subgraph computeConnectedSubgraph(
+            Map<Vertex, ArrayList<Vertex>> neighborsForVertex, Vertex startVertex) {
+        Subgraph subgraph = new Subgraph();
+        Queue<Vertex> q = new LinkedList<Vertex>();
+        q.add(startVertex);
+        while (!q.isEmpty()) {
+            Vertex vertex = q.poll();
+            for (Vertex neighbor : neighborsForVertex.get(vertex)) {
+                if (!subgraph.contains(neighbor)) {
+                    subgraph.addVertex(neighbor);
+                    q.add(neighbor);
+                }
+            }
+        }
+        return subgraph;
+//        if(subgraph.size()>1) return subgraph;
+//        return null;
+    }
+
+    private static void WriteNodesInSubGraph(
```

Comment:
Method name starts with a capital letter.

Reply:
Ah ok ... some old code there. BTW, are there some docs about recommended code formatting? I have not paid any attention to tabs, as the code occasionally seems to include them.

Reply:
https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/docs/Codestyle.md I think this is up-to-date.